blueprints/emeis-store/files/__root__/services/__name__.js

```diff
@@ -0,0 +1,5 @@
+import Store from "@ember-data/store";
+
+export default class <%= classifiedModuleName %>Store extends Store {
+  adapter = "<%= dasherizedModuleName %>";
+}
```

Comment:
I assume that `__name__` in the filename is the module name, right? If so, shouldn't the filename be something like `__name__-store` to match the classname?

Reply:
IIRC if you generate a component in a non-pod app, the file name ist `components/users.js` or `components/users/index.js`.

This blueprint has no pod support for now.

Reply:
Ah, I missunderstood the blueprint, NVM